browser/admin/admintemplate.html

```diff
@@ -61,6 +61,7 @@
 <body>
     <!--%BRANDING_JS%-->
     <script>
+        window.accessToken = '%ACCESS_TOKEN%';
```

Comment:
Wouldn't it be better to name this `jwtAccessToken = '%JWT_ACCESS_TOKEN%`, to avoid confusing with the wopi access token?